### ISSUE TITLE
Leveling Feature (Poe Lurker).

### ISSUE THREAD

**Issue by C1rdec** [PathOfBuildingCommunity/PathOfBuilding#1607]
Hey guys, I'm the developer of [Poe Lurker](https://github.com/C1rdec/Poe-Lurker) and I've develop a feature that help players with their builds based on Path of Building Code. The player is able to import the PoB code into Poe Lurker and I'm building a timeline that show gems/items to show when the player can equip new stuff.

Here is the current state of this feature:

![2020-10-04_23-29-41](https://user-images.githubusercontent.com/5436436/95113372-5ca32900-0710-11eb-902e-0904265d01af.gif)

![2020-09-16_23-39-07](https://user-images.githubusercontent.com/5436436/95113517-9116e500-0710-11eb-9402-6076b8fc07b9.gif)

I'd would like to know if it could be possible to add some kind of `metadata fields` on items/gems that would be exported in the PoB code. With this I would be able to group items by section.
Example: (`Leveling:10-20`, `leveling:20-30`, `endgame`)

Let me know if you guys needs further informations.
I'm more than happy to contribute but I want to know if this would make sense with you guys before I start coding.

**Comment by Nostrademous** [comment]
PoB already allows people to manage Passive Trees (so you can specify a tree at whatever increments you want).

Additionally, you can have gear/item sets (although socketed jewels, including cluster) are not supported. I had a [PR #334](https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/334) for this about 6 months ago but we were, and still are, waiting on better automated testing as it could break a lot of older builds.

**Comment by C1rdec** [comment]
> PoB already allows people to manage Passive Trees (so you can specify a tree at whatever increments you want).
> 
 Yeah for Passive Trees but it will be nice to have the same for gears/gems.

For example Zizaran is creating fake separator in the skills sections but there's no "official" way to group gear/gems by leveling phase.

**Comment by Wires77** [comment]
I think PoB's focus is more on optimizing an endgame build vs. a comprehensive game tool, and the team is hesitant to work on something that distracts from that goal, especially if it needs ongoing support.

I do like the notes idea though, as that is flexible enough to allow users to add to their build in their own way.  E.g. a recent discussion was around adding levels and quests at which you could complete certain gem setups, and having notes field here might let tools like PoE Lurker fill those out and present the build back to the user.

**Comment by C1rdec** [comment]
PoB is the first tool that new player come across since all the main streamers are using it.  The majority are trying to explain the leveling process with some work around (Like the one I linked). Having the notes section would let the ability to streamer to set the metadata info that Poe Lurker would know to display leveling section. 



> E.g. a recent discussion was around adding levels and quests at which you could complete certain gem setups, and having notes field here might let tools like PoE Lurker fill those out and present the build back to the user.

Poe Lurker already have all that informations.
![image](https://user-images.githubusercontent.com/5436436/94772754-5156a300-0388-11eb-8ab1-550e4601a482.png)


Like I said I'm more than happy to contribute but I just want to be sure that it's something that would be "approved".
***(Notes section @ skills level)***

**Comment by Wires77** [comment]
Yeah, go ahead with a PR, this sounds like a useful addition!

**Comment by ppoelzl** [comment]
Closed due to inactivity.
